src/extension/background-script/actions/ln/sendPayment.ts

```diff
@@ -33,13 +33,18 @@ export default async function sendPayment(
     response = await connector.sendPayment({
       paymentRequest,
     });
+
+    if (!response.data.preimage) {
```

Comment:
Remove this check? If there is an error during sendPayment an exception should be thrown on connector level.

Reply:
removed